app/models/PaymentMethod.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import models.{EnumerableType, Radioable, WithName}
+
+sealed trait PaymentMethod extends Radioable[PaymentMethod] {
+  val code: String
+
+  override val messageKeyPrefix: String = PaymentMethod.messageKeyPrefix
+}
+
+object PaymentMethod extends EnumerableType[PaymentMethod] {
```

Comment:
Is this needed? We're trying to gradually move away from this approach. Could ConsignmentTransportChargesPage just be a ReadOnlyPage[String] with "code" appended to the path? If all we're interested in is whether it's been answered or not